app/javascript/components/TestimonialSelectModal.tsx

```diff
@@ -82,42 +82,54 @@ export const TestimonialSelectModal = ({
       onClose={onClose}
       title="Insert reviews"
       footer={
-        <>
-          <Button onClick={onClose}>Cancel</Button>
-          <Button color="primary" onClick={() => onInsert(selectedReviewIds)}>
-            Insert
-          </Button>
-        </>
+        state.reviews.length > 0 ? (
+          <>
+            <Button onClick={onClose}>Cancel</Button>
+            <Button color="primary" onClick={() => onInsert(selectedReviewIds)}>
+              Insert
+            </Button>
+          </>
+        ) : (
+          <Button onClick={onClose}>Close</Button>
+        )
       }
     >
       <div>
-        <div className="flex flex-row items-center gap-2">
-          <input
-            type="checkbox"
-            role="checkbox"
-            checked={selectedReviewIds.length === state.reviews.length && state.reviews.length > 0}
-            onChange={toggleSelectAll}
-            aria-label="Select all reviews"
-          />
-          <p>Select all</p>
-        </div>
-        <section className="paragraphs" style={{ marginTop: "var(--spacer-2)" }}>
-          {state.reviews.map((review) => (
-            <SelectableReviewCard
-              key={review.id}
-              review={review}
-              isSelected={selectedReviewIds.includes(review.id)}
-              onSelect={() => toggleReviewSelection(review.id)}
-            />
-          ))}
-          {hasMorePages ? (
-            <div className="mt-4">
-              <Button onClick={handleLoadMore} disabled={isLoading}>
-                {isLoading ? "Loading..." : "Load more"}
-              </Button>
+        {state.reviews.length > 0 ? (
+          <>
+            <div className="flex flex-row items-center gap-2">
+              <input
+                type="checkbox"
+                role="checkbox"
+                checked={selectedReviewIds.length === state.reviews.length && state.reviews.length > 0}
+                onChange={toggleSelectAll}
+                aria-label="Select all reviews"
+              />
+              <p>Select all</p>
             </div>
-          ) : null}
-        </section>
+            <section className="paragraphs" style={{ marginTop: "var(--spacer-2)" }}>
+              {state.reviews.map((review) => (
+                <SelectableReviewCard
+                  key={review.id}
+                  review={review}
+                  isSelected={selectedReviewIds.includes(review.id)}
+                  onSelect={() => toggleReviewSelection(review.id)}
+                />
+              ))}
+              {hasMorePages ? (
+                <div className="mt-4">
+                  <Button onClick={handleLoadMore} disabled={isLoading}>
+                    {isLoading ? "Loading..." : "Load more"}
+                  </Button>
+                </div>
+              ) : null}
+            </section>
+          </>
+        ) : (
+          <div className="py-8 text-center">
+            <p>No reviews to insert</p>
+          </div>
```

Comment:
```suggestion
            <p>No reviews to insert</p>
```

I don't think we need the styling, wdyt?

Reply:
going with this https://github.com/antiwork/gumroad/pull/1341#issuecomment-3333780241